index.bs

```diff
@@ -4569,13 +4569,14 @@ credentials, then the sample code for performing such an authentication might lo
 
 <pre class="example" highlight="js">
     if (!window.PublicKeyCredential) { /* Platform not capable. Handle error. */ }
-
+  
+    var idBuffer = encoder.encode("agent007_4124732214")
```

Comment:
I think I'd prefer this to be something like

```
var idBuffer = new Uint8Array([183, 148, 245 /* more random bytes previously generated by the authenticator */]);
```

analogous with the `challenge` example, so as to not indicate that the credential ID is a readable identifier chosen by the RP or user.

Reply:
I concur with @emlun's comment above, and would even add an additional comment along the lines of "credential ID ought to be an opaque and random"